backend/_example/memory_store/go.mod

```diff
@@ -4,11 +4,12 @@ go 1.12
 
 require (
 	github.com/go-pkgz/jrpc v0.1.0
-	github.com/go-pkgz/lgr v0.6.3
+	github.com/go-pkgz/lgr v0.7.0
```

Comment:
I bumped a couple of modules because I can.

Disclaimer: remark/backend v1.5.0 wouldn't work without `replace` in `go.mod`, because of #589. But leaving v1.4.0 feels wrong.

Reply:
Moved to #628.